### PR TITLE
docs(integration): Add `LiteLLM` example to `/guides/integrations/`

### PR DESCRIPTION
**Pull request by ash0ts**
Added an example identical to `anthropic` and `openai` for `LiteLLM`

### CODE REVIEW

File: weave/weave_client.py [hunk 1]
```diff
@@ -489,7 +489,8 @@ def finish_call(
             summary = sum_dict_leaves([child.summary or {} for child in call._children])
         elif isinstance(output, dict) and "usage" in output and "model" in output:
             summary["usage"] = {}
-            summary["usage"][output["model"]] = {"requests": 1, **output["usage"]}
+            model_key = str(output["model"])  # Convert ObjectRef to string
```
Comment (tssweeney) [inline weave/weave_client.py:492]: Let's work on a different fix here: https://github.com/wandb/weave/pull/1891

**Review by tssweeney — CHANGES_REQUESTED** [review]
Let's work on a different fix here: https://github.com/wandb/weave/pull/1891

File: weave/weave_client.py (hunk 1 above)
Comment (ash0ts) [reply weave/weave_client.py]: Reverted my changes to allow for the changes in #1891

**Comment by ash0ts** [comment]
@tssweeney Merged the fixes

**Review by tssweeney — APPROVED** [review]

**Pull request merged**